site/src/api/api.ts

```diff
@@ -489,15 +489,23 @@ export const postWorkspaceBuild = async (
 export const startWorkspace = (
   workspaceId: string,
   templateVersionID: string,
+  debug = false,
```

Comment:
nit: It partially violates the open-closed principle. You may want to consider having a `log_level` instead. In the future, we may want to enable a custom log level like TRACE or WARN, so this will have to be refactored.

Reply:
Good, I will do that. But just to learn more, why does it violates the open-closed?

Reply:
Refactor done!